test/baselines.js

```diff
@@ -24,6 +24,8 @@ function build(file) {
 }
 
 describe('baselines', () => {
+  let rebaseline = !!process.env.npm_config_update_baselines;
```

Comment:
does this still work in npm 7? i'm not sure this is something to rely on, as opposed to `cross-env ENV_VAR=value npm run test-baselines`

Reply:
It does, yes. It's a [documented feature](https://docs.npmjs.com/cli/v7/using-npm/config#command-line-flags); it seems fine to rely on to me - better than relying on a new external dependency, certainly.

Reply:
cross-env is ancient and universal, but if it’s documented then that’s clearly fine.